### ISSUE TITLE
Feature request: display the date when the day has changed

### ISSUE THREAD

**Issue by brabalan** [jorgenschaefer/circe#216]
The current display only shows the time on the right, with no reference to the date. It would be nice to display the date when the day has  changed. Something like:

```
> I should go to bed                                                        [23:44]
<someone> good night!                                                       [23:46]
*** Thursday, September 3, 2015
> Good morning                                                              [07:12]
```

The date could be displayed when something needs to be displayed on a new day.

**Comment by jorgenschaefer** [comment]
That's a very good idea, thank you!

**Comment by jorgenschaefer** [comment]
Fixed in 8583caa
